src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -427,11 +708,45 @@ export class PostgresStorageAdapter {
     const qs = `SELECT count(*) FROM $1:name ${wherePattern}`;
     return this._client.one(qs, values, a => +a.count);
   }
+
+  performInitialization({ SchemaController }) {
```

Comment:
I think this is too much access, the DB adapters shouldn't be allowed to see the entire SchemaController.